src/common/stats.js

```diff
@@ -8,6 +8,7 @@ Take the stat of an item at a given iLvL (for example, 300 Crit at 850 iLvL).
 To figure out how much crit that would have at 860 iLvL, you'd do: `300 * 1.15 ^ ((860 - 850) / 15) * 0.994435486 ^(860-850)` for 311 Crit.
 Once the stat at a given iLvL is found, the only variable that needs to be changed is `iLvL`
 */
+// can confirm this formula works if given ilevel/stat other than whatever the "base" is. - kfinch
```

Comment:
This would probably be better suited as a unit test than a comment.

Reply:
Really, I should link to a math textbook or something that shows the function is ... commutative? I dunno..

Reply:
You don't have to do anything, just stating that proving this with a unit test would be more reliable and (excuse me) useful than a comment. You wouldn't do any math in such a test just check that given baseItemLevel X, baseStat Y and itemLevel Z, it returns the right (precalculated) value, even if those values aren't "base".